backend/schema/validate.go

```diff
@@ -512,9 +512,16 @@ func validateIngressRequestOrResponse(scopes Scopes, n *Verb, reqOrResp string,
 	if opt, ok := fieldType.(*Optional); ok {
 		fieldType = opt.Type
 	}
+
+	if ref, err := ParseRef(fieldType.String()); err == nil {
+		if ref.Module != "" && ref.Module != module.Name {
+			return // ignores references to other modules.
+		}
+	}
```

Comment:
This introduces logic to not validate resp body types for external modules since we don't have the full schema here, just a single module. This follows the comments for `ValidateModule` 
```go
// ValidateModule performs the subset of semantic validation possible on a single module.
//
// It ignores references to other modules.
func ValidateModule(module *Module) error {
```

Reply:
Does this get validated in `ValidateSchema` instead now? I don't see that in this PR.

Reply:
Yeah I believe that code is here: https://github.com/TBD54566975/ftl/blob/main/backend/schema/validate.go#L137 and tested here: https://github.com/TBD54566975/ftl/blob/main/backend/schema/schema_test.go#L211-L214